demos/demo.offspot.yaml

```diff
@@ -14,3 +14,5 @@ demos:
   - ident: wikipedia-en
     alias: wikipedia
     name: Wikipedia Package
+  - ident: orange-demo
```

Comment:
This is going to be `https://orange-demo.demo.hotspot.kiwix.org/`. Is this what you want? You can add an `alias` if you want

Reply:
Thanks. Added orange as alias